src/lib/NativeMetaTransaction.sol

```diff
@@ -53,22 +55,21 @@ abstract contract NativeMetaTransaction is EIP712, Nonces {
         return returnData;
     }
 
+    /// @notice verify if the meta transaction is legit
+    /// @dev Performs some validity check and checks if the signature matches the hash struct
+    /// See EIP-712 for details about `_verifySig`
```

Comment:
the EIP-712 standard doesn't explicitly contain `_verifySig` details. 
Do you mean EIP712.sol?